### PR TITLE
Improvement in Position section

### PR DESCRIPTION
**Pull request by super-pirata**
The Wikipedia link was pointing to a paragraph instead of the whole page.
Short explanation added.

### CODE REVIEW

**Comment by AminHP** [comment]
Hi. Thank you for your contribution. I am kind of busy at this moment, but I reviewed your pull request and the changes you have made were cool. However, there are some small issues, and I will comment on them in a few days.

File: README.ipynb [hunk 1]
```diff
@@ -37,7 +37,9 @@
     "So after months of work, I finally found out that these actions just make things complicated with no real positive impact. In fact, they just increase the learning time and an action like **Hold** will be barely used by a well-trained agent because it doesn't want to miss a single penny. Therefore there is no need to have such numerous actions and only `Sell=0` and `Buy=1` actions are adequate to train an agent just as well.\n",
     "\n",
     "### Trading Positions\n",
-    "If you're not familiar with trading positions, refer [here](https://en.wikipedia.org/wiki/Position_(finance)#Net_position). I'm not going to explain it but it's a very important concept and you should learn it as soon as possible.\n",
+    "If you're not familiar with trading positions, refer [here](https://en.wikipedia.org/wiki/Position_(finance)). It's a very important concept and you should learn it as soon as possible.\n",
```
Comment (AminHP) [inline README.ipynb]: It was my mistake that I didn't use backslash before the perantheses in the link. Please change `refer [here](https://en.wikipedia.org/wiki/Position_(finance))` to `refer [here](https://en.wikipedia.org/wiki/Position_\(finance\))`.

File: README.ipynb [hunk 2]
```diff
@@ -37,7 +37,9 @@
...
+    "\n",
+    "In a simple vision: **Long** position wants to buy shares when prices are low and proffit by sticking with them while their value is going up, and **Short** position wants to sell shares with high value and use this value to buy shares at a lower value, keeping the difference as proffit.\n",
```
Comment (AminHP) [inline README.ipynb]: Misspellings:
* Improovement -> Improvement (In the commit message)
* proffit -> profit

**Comment by AminHP** [comment]
Sorry for the delay. The explanation on the trading positions was very clear, I commented the problems. Please fix them. Thanks!

**Comment by super-pirata** [comment]
> 
> 
> Sorry for the delay. The explanation on the trading positions was very clear, I commented the problems. Please fix them. Thanks!

Fixed them! Thanks for the tips

**Pull request closed without merging**